cluster-bootstrap/src/main/resources/reference.conf

```diff
@@ -29,8 +29,8 @@ akka.management {
       service-name = "<service-name>"
 
       # The portName passed to discovery. This should be set to the name of the port for Akka Management
-      # If set to "" None is passed
-      port-name = "management"
+      # If set to "" None is passed and ${akka.management.http.port} is assumed.
+      port-name = ""
```

Comment:
Ah i see from the other PR now.

Reply:
But won't we then get back pods that match the label selector that don't have akka management enabled? This could result in some confusing connection errors. Perhaps we filter down below, will see